src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultFailoverPlugin.java

```diff
@@ -14,7 +15,7 @@
   public DefaultFailoverPlugin() {}
 
   @Override
-  public void init(PropertySet propertySet, HostInfo hostInfo, IFailoverPlugin next, Log log) {
+  public void init(Connection connection, PropertySet propertySet, HostInfo hostInfo, IFailoverPlugin next, Log log) {
```

Comment:
Question: why is this parameter added, but not used or accessed in the code?

Reply:
From the interface, one of the other plugins needed it. Would another way to overload the init() or create another interface?

Reply:
If another piece of code needs it, it's completely ok - it just wasn't clear to see it from this PR